src/pages/VenueLandingPage/VenueLandingPage.tsx

```diff
@@ -10,11 +10,9 @@ import {
   IFRAME_ALLOW,
 } from "settings";
 
-import { VenueEvent } from "types/venues";
 import { VenueAccessMode } from "types/VenueAcccess";
 
 import { hasUserBoughtTicketForEvent } from "utils/hasUserBoughtTicket";
-import { WithId } from "utils/id";
 import { isUserAMember } from "utils/isUserAMember";
```

Comment:
Looks like `utils/isUserAMember` / `isUserAMember` may still need to be cleaned up/removed from the codebase?

Reply:
fixed in ebe967cd50437f138f6685ec3ce64daef7abeb40